backend/controllers/user.js

```diff
@@ -117,7 +117,7 @@ module.exports.getAllUsers = async (req, res) => {
 
 module.exports.changePassword = async (req, res) => {
   try {
-    const user = await User.findByPk(req.user.id);
```

Comment:
This should be user, the person who is changing their password should be logged in.